turbo.json

```diff
@@ -39,10 +39,8 @@
     "dev": {
       "cache": false
     },
-    "@faststore/core#dev": {
-      "dependsOn": [
-        "@faststore/api#build"
-      ]
+    "dev:server": {
```

Comment:
you need to add the `dev:graphql`, `build:cjs` and `build:esm` here too

Reply:
Isn't turbo's --no-cache enough?

Reply:
I usually run them using yarn: `yarn workspace @faststore/core dev:graphql`

Reply:
you can improve your experience using only turbo because it will optimize calls with caches and for this to work, the scripts need to be mapped in advance.

I discourage you from stopping using yarn and only using it to install dependencies.

Reply:
I didn't know that! I'll try to run with turbo and see if it works, tks

Reply:
[done](https://github.com/vtex/faststore/pull/2150/commits/34fe8f0e9a836d1eadda9a6fe2b30ed52e24d217)!